group/lc/outputs.tf

```diff
@@ -0,0 +1,9 @@
+# Outputs
+
+output "sg_id" {
+  value = "${aws_security_group.sg_asg.id}"
+}
+
+output "lc_id" {
+  value = "${coalesce(aws_launch_configuration.lc.id, aws_launch_configuration.lc_ebs.id)}"
```

Comment:
Too bad it didn't work. In reviewing the state file, this output was just not being set at all. I was able to use that however to find a solution. I'm double declaring the output knowing that only one will get set and that is working.